crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -82,6 +88,86 @@ struct TxnSummary {
     events: Vec<Event>,
 }
 
+static GENESIS: Lazy<Genesis> = Lazy::new(create_genesis_module_objects);
```

Comment:
Note that this whole block was moved from `sui-adapter/src/genesis.rs` without changes.